web/src/main/java/nl/tehdreamteam/se42/web/soap/SoapWebService.java

```diff
@@ -15,6 +17,8 @@
  */
 public class SoapWebService implements Service {
 
+    private final Logger logger = LogManager.getLogger(getClass().getSimpleName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make the "logger" logger private static final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1312)